test/e2e/iam/project_settings_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (iam && atlas)
```

Comment:
I cannot see this test being ran, are you sure these are the right tags? are you planning to create this iam one? I think for IAM tests you should refer to the evergreen.yml:

```
  - name: atlas_iam_e2e
    tags: ["e2e","generic","atlas"]
```

so this should probably  be `//go:build e2e || (atlas && generic)`

Reply:
I updated the e2e test flags to `iam`, I checked and Evergreen runs the tests correctly.